hack/move-upstream.sh

```diff
@@ -18,6 +18,8 @@ repo="${UPSTREAM_REPO:-k8s.io/kubernetes}"
 package="${UPSTREAM_PACKAGE:-pkg/api}"
 
 patch="${TMPDIR}/patch"
+rm -rf "${patch}"
```

Comment:
I don't see us forcing `TMPDIR` to have a value.  If it doesn't, this causes you force delete `/patch`, which seems naughty, though not outright wrong.